recipes/django-material/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "django-material" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "5ad516f55f05e2fabb31797d4c5ae5819ae37c8945386d947f09eb550bf4463b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+
+test:
+  run_test.py
```

Comment:
I think you don't specify `run_test.py` here. The name is canonical so conda just picks it up if it's present.